handlers/node_api.py

```diff
@@ -392,3 +392,28 @@ def nc_blueprint_information(
         "body": json.dumps(response or {}),
         "headers": {"Content-Type": "application/json"},
     }
+
+
+@ApiGateway()
+def nc_blueprint_source_code(
+    event: ApiGatewayEvent, _context: LambdaContext, node_api: Optional[NodeApi] = None
+) -> dict:
+    """Get blueprint source code."""
+    node_api = node_api or NodeApi()
+    blueprint_id = event.query.get("blueprint_id")
+
+    if blueprint_id is None:
+        raise ApiError("invalid_parameters")
+
+    # This might throw HathorCoreTimeout error
+    response = node_api.get_nc_blueprint_source_code(blueprint_id)
+
+    if response is None or "error" in response:
+        message = response.get("error") if (response and "error" in response) else ""
```

Comment:
question(non-blocking): Should we add a default error message here? It may help with eventual debuggings

```suggestion
        message = response.get("error") if (response and "error" in response) else "Unknown error when retrieving blueprint source code"
```

Reply:
Done https://github.com/HathorNetwork/hathor-explorer-service/pull/333/commits/39a0278b95df3e7bea194618910b917022fae587